### ISSUE TITLE
[Feature request] Add parking option after navigation

### ISSUE THREAD

**Issue by annasoloviova** [osmandapp/Osmand#3329]
Request from our user: add menu 'remember my current position as parking 
position' to the dialogue that appears at the end of navigation.

**Comment by hakuchi** [comment]
why you not use "OsmAnd - Parking Plugin"?
https://play.google.com/store/apps/details?id=net.osmand.parkingPlugin

**Comment by vshcherb** [comment]
Good point. This is a request to use Parking plugin and to add a shortcut how to set a parking point.

**Comment by breversa** [comment]
Yes, it'd be grat to have the 'set parking emplacement" in the end of navigation dialog.

So far, it offers to *look for* a parking place, but not set the current emplacement as parking emplacement.
